_docs/backenders/strategy-2021/roles/business-development-proposals.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: default
+title: Business Development Proposals Role
+parent: Roles
+grand_parent: Strategy 2021
+---
+
+# Business Development Proposals role definition
+
+A close collaboration of the product teams with the Business Development team
+is essential for Vizzuality to pursue high impact new opportunities, to develop
+compelling proposals and to improve business retention through new phases of
+projects, partnerships and referrals.
+
+This role is the point of contact between the Backend and Business Development
+functional areas *when collaborating on opportunities and proposals*.
+
+Liaising with the Business Development team in terms of projected capacity needs
+for proposals awaiting evaluation/decision by potential clients is not part of
+this role.
+
+**Current responsible**: N/A
+
+## Responsibilities
+
+- Review Copper notifications
+
+- Proactively interface with the BD team to assess if active involvement is needed for:
+  - evaluating new leads
+  - researching and crafting together each new proposal the team decides to
+    develop
+
+- Ensure a backend engineer will collaborate on each proposal (either actively
+  participating in researching/crafting it for larger, strategic proposals, or
+  at least cross-checking a late draft for smaller/routine proposals)
+
+- Ensure (either directly or by seeking help from a fellow BE engineer) that
+  each proposal includes adequate budget for baseline advisory/support backend
+  work if substantive backend work is not part of a proposal
+
+- Ensure that the backend group is collaborating actively and efficiently in
+  business development process (see "Bernat's Dekalog" document)
+
+- Ensure that the business development team is up to date on the backend
+  strategy so that they can pursue leads that match capabilities and
+  ongoing direction of the team
+
+- Review the BD financial dashboard
+
+  *This may be more relevant to the Capacity role, in terms of promised
```

Comment:
I do agree that in this point capacity rol will need fluent info with the bd representative in order to be able to evaluate and be able to forecast the capacity.

Reply:
@aagm should we move this responsibility to the Capacity role then? Or we can leave this as is - it doesn't hurt to review the financial dashboard periodically in any case, and we can adjust/remove this later if not really relevant.

Reply:
Leave as it is but noting the need for fluid information between both roles